tagsdoc/src/main/resources/com/sun/tlddoc/resources/tag.html.xsl

```diff
@@ -3,6 +3,7 @@
 <!--
   - <license>
   - Copyright (c) 2003-2004, Sun Microsystems, Inc.
+  - Copyright (c) 2021-2022 Contributors to the Eclipse Foundation
```

Comment:
Shouldn't this just be `Copyright (c) 2022-2022 Contributors to the Eclipse Foundation` ?